src/adfh/ADFH.h

```diff
@@ -174,7 +174,7 @@
 	Prototypes for Interface Routines
 ***********************************************************************/
 
-#if defined(_WIN32) && defined(BUILD_DLL)
+#if defined(WIN32) && defined(BUILD_DLL)
```

Comment:
I think _WIN32 is correct. Why use WIN32?

Reply:
Indeed, `_WIN32` is correct. CMake sets `WIN32` for legacy reasons, but the real one should be preferred.